project/resources/serializers.py

```diff
@@ -4,11 +4,32 @@
 from tagging.serializers import TagSerializer, TagsSerializerField
 
 
+class MediaTypeSerializerField(serializers.ChoiceField):
+
+    def to_representation(self, value):
+
+        valid_media_types = ', '.join(item for item in self.choices)
+
+        if not value:
+            return ''
+
+        else:
+            try:
+                media_type = self.choices[value]
+
+            except KeyError as err:
```

Comment:
One question here: when I passed in an invalid `media_type` field, I was hoping that the key error would say "Invalid media type. The media type should be one of the following: [list]`, but instead the error from Postman is more vague: 
![image](https://user-images.githubusercontent.com/4512699/80894860-e77c8500-8c93-11ea-8565-1f5ba4069d7a.png)

Do you know why?

Reply:
_interesting_.  It did so for me, but I also POSTED a full record:

![image](https://user-images.githubusercontent.com/5923094/80895052-94a3cd00-8c95-11ea-81d5-d7e7bc234dc0.png)

Reply:
I see that now, after pulling in this merged branch in `master`! I wonder if maybe I needed to do `docker-compose down` and re-run it 🤷Anyway, solved now :)